baybe/utils/basic.py

```diff
@@ -88,3 +88,15 @@ def group_duplicate_values(dictionary: dict[_T, _U]) -> dict[_U, list[_T]]:
     for key, value in dictionary.items():
         group.setdefault(value, []).append(key)
     return {k: v for k, v in group.items() if len(v) > 1}
+
+
+def to_tuple(x: Sequence) -> tuple:
+    """Convert any sequence into a tuple.
+
+    This wrapper is only used to avoid mypy warnings for attrs converters:
+    * https://github.com/python/mypy/issues/8417
+    * https://github.com/python/mypy/issues/8389
+    * https://github.com/python/mypy/issues/5313
+    """
+    # TODO: Remove wrapper once mypy support is there
```

Comment:
Can you add a reminder for this using our reminder functionality in the repo?

Reply:
Would love to, but I'm not aware of any specific date when that would be fixed. Or do you suggest to just set a reminder for in 6months so? Could do, but then the question is: why here and not for all other TODOs where we wait on external features/fixes?

Reply:
Hmm... good argument. Flooding the repository with reminders without specific dates does not sound like a good idea, so I am happy with not having a reminder here.